opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbanking/impl/config/hibernate/PrefixAndSnakeCasePhysicalNamingStrategy.java

```diff
@@ -0,0 +1,53 @@
+package de.adorsys.opba.tppbanking.impl.config.hibernate;
+
+import com.google.common.base.CaseFormat;
+import org.hibernate.boot.model.naming.Identifier;
+import org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl;
+import org.hibernate.engine.jdbc.env.spi.JdbcEnvironment;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Converts Hibernate snakeCase naming into Database-friendly camel_case, also adds prefix to the tables.
+ */
+@Configuration
+public class PrefixAndSnakeCasePhysicalNamingStrategy extends PhysicalNamingStrategyStandardImpl {
```

Comment:
My IHMO, that if we are going to have separate module for database management - liquibase migration, etc - this file should be there

Reply:
ok. that will be next step